src/pages/timecard/Timecard.jsx

```diff
@@ -154,22 +154,31 @@ const updateTimeEntryContextData = async (
   validateServiceErrors(setServiceError, async () => {
     const timeEntriesResponse = await getTimeEntries(timeEntriesParams);
 
+    const timeCardStart = dayjs(date).startOf('day').add(1, 'minute');
+    const timeCardEnd = dayjs(date).endOf('day');
+
     if (timeEntriesResponse.data.items?.length > 0) {
-      const existingTimeEntries = timeEntriesResponse.data.items.map(
+      const filteredTimeEntries = timeEntriesResponse.data.items.filter(
```

Comment:
if we revisit the filter after a while it might not be easy to workout why we are doing this specific filtering, maybe we should update this variable name to indicate what's actually been filtered.
Maybe something that represents "time entries that span onto date".

Reply:
resolving as the additional filtering has now been removed.